pkg/alexa/l10n/l10n_test.go

```diff
@@ -61,17 +54,35 @@ var enUS = &l10n.Locale{
 	},
 }
 
+func TestDefaultRegistry(t *testing.T) {
+	// make sure we have an empty default
+	l10n.DefaultRegistry = l10n.NewRegistry()
+
+	err := l10n.Register(deDE)
+	assert.NoError(t, err)
```

Comment:
You test should have the form:
```
<Setup>

<Test>

<Assertion>
```

Like this i dont know what is setup and what is test.

Reply:
Do you mean putting all the variable assignments in one block and below have all the assertions in one block, or really splitting completely, like in this case having `l10n.Register` test in a separate function than `l10n.Resolve`?

Reply:
No, i mean separate with space. Like this https://github.com/hamba/pkg/blob/master/stats/aggregate_test.go#L20. It is clear what is setup, what is the test and what is the assertion.